tests/suites/test_suite_x509parse.data

```diff
@@ -911,6 +911,14 @@ X509 CRT verification #97 (next profile Valid Cert SHA256 Digest)
 depends_on:MBEDTLS_SHA256_C:MBEDTLS_PEM_PARSE_C:MBEDTLS_RSA_C:MBEDTLS_PKCS1_V15:MBEDTLS_ECDSA_C:MBEDTLS_SHA1_C
 x509_verify:"data_files/cert_sha256.crt":"data_files/test-ca.crt":"data_files/crl-ec-sha256.pem":"NULL":0:0:"next":"NULL"
 
+X509 CRT verification #98 (Revoked Cert, revocation date in the future)
```

Comment:
It took me some time to notice the difference between the two tests, could you perhaps append `HAVE_TIME_DATE`/`!HAVE_TIME_DATE` to the description in order to make it more apparent?

Reply:
Generally, using numbers in test cases is deprecated unless it's for tests that do exactly the same thing with different (pseudo)random data.

I should get around to writing this guide on writing test case descriptions.